src/clientManager.js

```diff
@@ -20,6 +20,19 @@ ClientManager.prototype.findClient = function(name) {
 	return false;
 };
 
+ClientManager.prototype.writeUser = function(userDict) {
+	var users_path = Helper.HOME + "/users" ;
+	var user_path = users_path + "/" + userDict.user + ".json";
+
+	mkdirp(users_path);
```

Comment:
Should we set correct chmod here too? https://www.npmjs.com/package/mkdirp#mkdirpdir-opts-cb